proto/contimage/contimage.proto

```diff
@@ -0,0 +1,53 @@
+syntax = "proto3";
+
+option go_package = "github.com/DataDog/agent-payload/v5/contimage";
+
+package datadog.contimage;
+
+// ContainerImagePayload represents the main container image payload
+message ContainerImagePayload {
+    string version = 1; // version represents the ContainerImagePayload message version
+    string host = 2; // host contains the hostname
+    repeated ContainerImage images = 3; // Images metadata
+    // repeated VulneratibilityReport reports = 4; // vulneratibility reports
+}
+
+// ContainerImage contains the details of a container image
+message ContainerImage {
+    string id = 1;
+    string registry = 2;
+    string name = 3;
+    repeated string tags = 4;
+    string digest = 5;
+    int64 size = 6;
+    string repo_digest = 7;
+
+    // # platform info
+    // message OperatingSystem {
+    //   string name = 1;
+    //   string version = 2;
+    //   string architecture = 3;
+    // }
```

Comment:
IMO this can be uncommented

Reply:
Done: https://github.com/DataDog/agent-payload/pull/196/commits/1e642885f7f4dfb80c59d4e0d186af794b848b59